cobalt/build/configs/android-cobalt-arm/args.gn

```diff
@@ -1,4 +1,2 @@
 target_os = "android"
 target_cpu = "arm"
-
-is_cobalt = true
```

Comment:
Add `is_cobalt = false` to the equivalent non-cobalt configs, i.e. https://github.com/youtube/cobalt/blob/main/cobalt/build/configs/android-arm/args.gn (there should be one for each of the cobalt configs)

Reply:
done